xdcs-server/src/main/java/pl/edu/agh/xdcs/services/TaskService.java

```diff
@@ -162,13 +167,35 @@ public TaskCreationWizard addResourcePattern(
             return this;
         }
 
+        public TaskCreationWizard addAllKernelArgument(List<String> kernelArguments) {
+            this.kernelArguments = new LinkedList<>();
+            for (String kernelArgument : kernelArguments) {
+                String objectId = objectRepository.store(Blob.fromBytes(kernelArgument.getBytes()));
```

Comment:
tego trochę nie rozumiem, zapisujesz argument będący stringiem z domyślnym kodowaniem platformy do OR?

Reply:
to była wersja w trakcie developmentu, poprawione

Reply:
a wrzuciłeś poprawkę?

Reply:
poprawione